packages/contracts/src/make-dump.ts

```diff
@@ -89,5 +89,27 @@ export const makeStateDump = async (cfg: RollupDeployConfig): Promise<any> => {
     }
   }
 
-  return dump
+  return {
+    config: {
+      chainId: 420,
+      homesteadBlock: 0,
+      eip150Block: 0,
+      eip155Block: 0,
+      eip158Block: 0,
+      byzantiumBlock: 0,
+      constantinopleBlock: 0,
+      petersburgBlock: 0,
+      istanbulBlock: 0,
+      muirGlacierBlock: 0,
+      clique: {
+        period: 0,
+        epoch: 30000,
+      },
+    },
+    difficulty: '1',
+    gasLimit: '11000000',
+    extradata:
+      '0x000000000000000000000000000000000000000000000000000000000000000000000398232E2064F896018496b4b44b3D62751F0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000',
```

Comment:
This extradata makes 0x00000398... (address listed above) the only clique signer.